### ISSUE TITLE
v3.0.5 breaks Symfony Maker Command - Syntax error, unexpected T_NAME_FULLY_QUALIFIED

### ISSUE THREAD

**Issue by MKHSKY** [zircote/swagger-php#841]
Since Version 3.0.5 we cannot use the symfony maker command to regenerate entities anymore.

**Error Message**
```
In ParserAbstract.php line 315:
                                                             
  [PhpParser\Error]                                          
  Syntax error, unexpected T_NAME_FULLY_QUALIFIED on line 3  
                                                             

Exception trace:
  at /workspaces/php-parser-error/src/vendor/nikic/php-parser/lib/PhpParser/ParserAbstract.php:315
 PhpParser\ParserAbstract->doParse() at /workspaces/php-parser-error/src/vendor/nikic/php-parser/lib/PhpParser/ParserAbstract.php:158
 PhpParser\ParserAbstract->parse() at /workspaces/php-parser-error/src/vendor/symfony/maker-bundle/src/Util/ClassSourceManipulator.php:879
 Symfony\Bundle\MakerBundle\Util\ClassSourceManipulator->setSourceCode() at /workspaces/php-parser-error/src/vendor/symfony/maker-bundle/src/Util/ClassSourceManipulator.php:73
 Symfony\Bundle\MakerBundle\Util\ClassSourceManipulator->__construct() at /workspaces/php-parser-error/src/vendor/symfony/maker-bundle/src/Doctrine/EntityRegenerator.php:209
 Symfony\Bundle\MakerBundle\Doctrine\EntityRegenerator->createClassManipulator() at /workspaces/php-parser-error/src/vendor/symfony/maker-bundle/src/Doctrine/EntityRegenerator.php:75
 Symfony\Bundle\MakerBundle\Doctrine\EntityRegenerator->regenerateEntities() at /workspaces/php-parser-error/src/vendor/symfony/maker-bundle/src/Maker/MakeEntity.php:779
 Symfony\Bundle\MakerBundle\Maker\MakeEntity->regenerateEntities() at /workspaces/php-parser-error/src/vendor/symfony/maker-bundle/src/Maker/MakeEntity.php:135
 Symfony\Bundle\MakerBundle\Maker\MakeEntity->generate() at /workspaces/php-parser-error/src/vendor/symfony/maker-bundle/src/Command/MakerCommand.php:104
 Symfony\Bundle\MakerBundle\Command\MakerCommand->execute() at /workspaces/php-parser-error/src/vendor/symfony/console/Command/Command.php:258
 Symfony\Component\Console\Command\Command->run() at /workspaces/php-parser-error/src/vendor/symfony/console/Application.php:934
 Symfony\Component\Console\Application->doRunCommand() at /workspaces/php-parser-error/src/vendor/symfony/framework-bundle/Console/Application.php:96
 Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() at /workspaces/php-parser-error/src/vendor/symfony/console/Application.php:264
 Symfony\Component\Console\Application->doRun() at /workspaces/php-parser-error/src/vendor/symfony/framework-bundle/Console/Application.php:82
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /workspaces/php-parser-error/src/vendor/symfony/console/Application.php:140
 Symfony\Component\Console\Application->run() at /workspaces/php-parser-error/src/bin/console:43

make:entity [-a|--api-resource] [--regenerate] [--overwrite] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command> [<name>]
```

**Steps to reproduce**
- Clone the repository https://github.com/MKHSKY/zircote-swagger-php-issue-841
- Run the VSCode devcontainer
- Run 
   ``` bash
   cd src && composer update && bin/console make:entity --regenerate
   ```

**Steps to start the devcontainer**
- You need Docker installed
- You need vscode
- You need this vscode Extension: https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack
- After that, open the repo -> Press F1 -> "Rebuild and Reopen in Container"
- Open the terminal

please note all the [commits](https://github.com/MKHSKY/zircote-swagger-php-issue-841/commits/master), you can see the difference

**Comment by DerManoMann** [comment]
this is already fixed on master.
@bfanger if you could find the time for a follow up release and packagist update that would be great

**Comment by bfanger** [comment]
Sure! i'll tag it 3.1.0, like you proposed earlier,  @DerManoMann feel free to create tags when you think a feature is ready.

**Comment by bfanger** [comment]
Released 3.1.0

@DerManoMann I've also added you as a maintainer, you should also be able to press the  "Update" button on the https://packagist.org/packages/zircote/swagger-php to sync the changes from GitHub to Packagist.
